packetbeat/_meta/fields.yml

```diff
@@ -1748,3 +1748,293 @@
             If the call resulted in exceptions, this field contains the exceptions in a human
             readable format.
 
+- key: tls
+  title: "TLS"
+  description: >
+    TLS-specific event fields.
+  fields:
+    - name: tls
+      type: group
+      fields:
+        - name: handshake_completed
+          type: boolean
+          description: >
+            Whether the TLS negotiation has been successful and the session has
+            transitioned to encrypted mode.
+
+        - name: resumed
+          type: boolean
+          description: >
+            If the TLS session has been resumed from a previous session.
+
+        - name: resumption_method
+          type: keyword
+          description: >
+            If the session has been resumed, the underlying method used. One of
+            "id" for TLS session ID or "ticket" for TLS ticket extension.
+
+        - name: client_certificate_requested
+          type: boolean
+          description: >
+            Whether the server has requested the client to authenticate itself
+            using a client certificate.
+
+        - name: client_hello
+          type: group
+          fields:
+            - name: version
+              type: keyword
+              description: >
+                The version of the TLS protocol by which the client wishes to
+                communicate during this session.
+
+            - name: timestamp
+              type: long
+              description: >
+                The current time and date in standard UNIX 32-bit format
```

Comment:
Is this seconds since Unix epoch? Either way I suggest converting it to a date rather than a long so that’s easier to use in ES.

Reply:
Right. Will do